core/variant/variant_parser.cpp

```diff
@@ -865,13 +865,47 @@ Error VariantParser::parse_value(Token &token, Variant &value, Stream *p_stream,
 			}
 
 			get_token(p_stream, token, line, r_err_str);
-			if (token.type != TK_NUMBER) {
+			// Permit empty RID.
+			if (token.type == TK_PARENTHESIS_CLOSE) {
+				value = RID();
+				return OK;
+			} else if (token.type != TK_NUMBER) {
 				r_err_str = "Expected number as argument";
```

Comment:
```suggestion
				r_err_str = "Expected number as argument or ')'";
```
Or similar but unsure if required here

Reply:
Yes, the message needs to be updated.